config/filter.d/nginx-botsearch.conf

```diff
@@ -0,0 +1,29 @@
+# Fail2Ban filter to match web requests for selected URLs that don't exist
+#
+
+[Definition]
+
+failregex = ^<HOST> \- \S+ \[\] \"(GET|POST) \/<block> \S+\" 404 .+$
+			^ \[error\] \d+#\d+: \*\d+ (\S+ )?\"\S+\" (failed|is not found) \(2\: No such file or directory\), client\: <HOST>\, server\: \S*\, request: \"(GET|POST) \/<block> \S+\"\, .*?$
+
+ignoreregex = 
+
+
+[Init]
+
+# Block is the actual non-found directories to block
+block = \/?(<webmail>|<phpmyadmin>|<wordpress>|cgi-bin|mysqladmin)[^,]*
```

Comment:
I am asking in https://github.com/fail2ban/fail2ban/pull/557/files#r23978582  on why we have decided to go the way of explicit fail2ban variables here instead of just relying on string interpolations... anyways -- since all those match with apache-botsearch (or am I missing difference) we should have now botsearch-common.conf which would define them all and then only get included into apache- and ngnix- to be used in failregex's